Makefile

```
@@ -368,6 +368,14 @@ deploy-dependencies: kustomize dependencies-manifests ## Deploy dependencies to
 	$(KUSTOMIZE) build config/dependencies | kubectl apply -f -
 	kubectl -n "$(KUADRANT_NAMESPACE)" wait --timeout=300s --for=condition=Available deployments --all
 
+.PHONY: install-metallb
+install-metallb: $(KUSTOMIZE)
```

minor:

you might want to add a description next to the target in order to appear in the help command

```suggestion
install-metallb: $(KUSTOMIZE) ## Installs MetalLB Load Balancer.
```

---

done